iphone/cli/commands/_build.js

```diff
@@ -2626,7 +2632,7 @@ iOSBuilder.prototype.createXcodeProject = function createXcodeProject(next) {
 						child.comment = '"' + appName + '.app"';
 					} else if (child.comment === 'Titanium.entitlements') {
 						child.comment = '"' + appName + '.entitlements"';
-					} else if (child.comment === 'LaunchScreen.storyboard' && appc.version.lt(this.xcodeEnv.version, '7.0.0')) {
+					} else if ((child.comment === 'LaunchScreen.storyboard' && appc.version.lt(this.xcodeEnv.version, '7.0.0')) || child.comment === 'ExternalAccessory.framework') {
```

Comment:
Not sure if that the `ExternalAccessory` change is related to this ticket. Let's not mix too many changes together if possible.